gui/app/batteryinfo.cpp

```diff
@@ -0,0 +1,6 @@
+#include "batteryinfo.h"
```

Comment:
Probably don't need a .cpp file for this much code - just make these inline functions in the class declaration